### ISSUE TITLE
[TF2] Console not opening after launch

### ISSUE THREAD

**Issue by Tintouin** [ValveSoftware/Source-1-Games#5303]
Whenever you launch the game pressing the console bind won't do anything even after changing it.
The only workarounds are launching the game with the `-console` parameter or launching a game on any server (it even works on training mode and MvM) to then make the console bind work. 
After this the bind works again until the game is closed.

Maybe some in game parameters attached to the engine makes the console able to be opened.

**Comment by NULLYUKI** [comment]
I would say this is the intended behavior, as this is also been written down on the "[Valve Developer Community](https://developer.valvesoftware.com/wiki/Developer_console#Enabling_the_console)" website.
![grafik](https://github.com/ValveSoftware/Source-1-Games/assets/42326194/97bb287e-458c-4445-961b-605fdaa50b16)

**Comment by JoriKos** [comment]
> I would say this is the intended behavior, as this is also been written down on the "[Valve Developer Community](https://developer.valvesoftware.com/wiki/Developer_console#Enabling_the_console)" website. ![grafik](https://user-images.githubusercontent.com/42326194/279841816-97bb287e-458c-4445-961b-605fdaa50b16.png)

I believe this is not what the user was referring to. They're talking about the console already being enabled through that setting, but being unable to do so when they open the game (only after being in a game first).

I have a bug similar to this sometimes, where I have to alt tab before being able to open the console. Not sure if that is related to this bug, but I don't experience this bug at all. I can open the console just fine after starting up the game.
